sycl/test-e2e/Basic/sampler/sampler.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: aspect-image
```

Comment:
I *think* this is the right one here (and not the legacy) but that's mostly a guess...

Reply:
`sampler` was SYCL 1.2.1, so they count as legacy.
```suggestion
// REQUIRES: aspect-ext_intel_legacy_image
```